packages/demos/src/server-side-rendering/webpack-config.js

```diff
@@ -5,34 +5,50 @@ const webpack = require('webpack');
 const {merge} = require('webpack-merge');
 const {webpackBaseConfig} = require('../webpack-base-config');
 
-/**
- * @type {webpack.Configuration}
- */
-const featureAppConfig = merge(webpackBaseConfig, {
-  entry: path.join(__dirname, './feature-app.tsx'),
-  externals: {
-    react: 'react',
-  },
-});
-
 /**
  * @type {webpack.Configuration[]}
  */
 const configs = [
-  merge(featureAppConfig, {
+  merge(webpackBaseConfig, {
+    entry: path.join(__dirname, './feature-app.tsx'),
     output: {
       filename: 'feature-app.umd.js',
       libraryTarget: 'umd',
       publicPath: '/',
     },
+    externals: {
+      react: 'react',
+    },
   }),
-  merge(featureAppConfig, {
+  merge(webpackBaseConfig, {
+    entry: {},
+    output: {
+      publicPath: '/',
+    },
+    plugins: [
+      new webpack.container.ModuleFederationPlugin({
+        filename: 'feature-app.federated.js',
+        name: '__feature_hub_feature_app_module_container__',
+        exposes: {
+          featureAppModule: path.join(__dirname, './feature-app'),
+        },
+        shared: {
+          react: {singleton: true},
```

Comment:
why is wmf not also configured for the integrator?

Reply:
fixed